std/machines/small_field/pointer_arith.asm

```diff
@@ -0,0 +1,35 @@
+/// Creates constraints that increments a pointer by one full 32-bit word.
```

Comment:
```suggestion
/// Creates constraints that increments a pointer.
```
(you always increment by the abstract "one" which doesn't have a size, only the pointer has a size)

Reply:
Oh wait, it increments them by the _size_ of a word?

Reply:
I increment by 4.

Could be an argument.